lib/workers/global/config/parse/file.ts

```diff
@@ -43,10 +45,7 @@ export async function getParsedContent(file: string): Promise<RenovateConfig> {
 }
 
 export async function getConfig(env: NodeJS.ProcessEnv): Promise<AllConfig> {
-  let configFile = env.RENOVATE_CONFIG_FILE ?? 'config.js';
-  if (!upath.isAbsolute(configFile)) {
-    configFile = `${process.cwd()}/${configFile}`;
-  }
```

Comment:
What does this acheive? If you only want to feed the relative path by itself this change doesn't make sense.

Reply:
Using a dynamic import on a file (not a module) requires an absolute path, i.e. `await import('relative/path/to/file.js')` doesn't work. Here, only the path of the `.js` config file passed via `RENOVATE_CONFIG_FILE` is converted to an absolute path. But it's also valid to pass a `.js` config file via CLI argument, and this path also needs conversion to an absolute path, which hasn't happened prior to this PR. Hence, I've removed the conversion logic downstream, so that any `.js` config file path &ndash; no matter how it was provided &ndash; gets converted to an absolute path.

Reply:
btw: typescript is compiling `import(...)` to `Promise.resolve(require(...))`

Reply:
In any case, I was getting this error. Feel free to reproduce it. This PR solves it for me.